utils/build/tsconfig.client.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.common.json",
+  "compilerOptions": {
+    "module": "esnext",
+    "jsx": "react"
+  },
+  "include": ["../../client", "../../utils"]
```

Comment:
you might need exclude for server directory here.

Reply:
I shall look into this